### PR TITLE
Feature/return recipe

### PR DESCRIPTION
**Pull request by aviquez96**

### CODE REVIEW

**Comment by aviquez96** [comment]
This is a sample POST request to test the service

http://localhost:8585/suggestion/5e7d5b4b3ea52245ea87faed

You have to run the suggestion, recipe and ingredient services :)

File: suggestion/src/controller/suggestRecipe.ts [hunk 1]
```diff
@@ -15,32 +16,48 @@ const PER_PAGE = 25;
...
             if (retRecipes.length === NUM_OF_RECIPES) {
-                // TODO return recipes instead of Hashes
+                // TODO: return recipes instead of Hashes
                 return retRecipes;
             }
         }
         pageCount++;
     }
+
     // TODO return recipes instead of Hashes
```
Comment (DConnorFong) [inline suggestion/src/controller/suggestRecipe.ts:61]: We have done this in the PR I assume. We can delete this comment

File: suggestion/src/controller/suggestRecipe.ts [hunk 2]
```diff
@@ -15,32 +16,48 @@ const PER_PAGE = 25;
...
+            if (compareHash(recipeBitString, hashIngredients) >= threshold) {
+                for (const object of Object.keys(recipeObject)) {
+                    if (recipeBitString === recipeObject[object].ingredients) {
+                        retRecipes.push(recipeObject[object].id);
+                    }
+                }
             }
             if (retRecipes.length === NUM_OF_RECIPES) {
-                // TODO return recipes instead of Hashes
+                // TODO: return recipes instead of Hashes
```
Comment (DConnorFong) [inline suggestion/src/controller/suggestRecipe.ts:54]: Also here

Comment (aviquez96) [reply suggestion/src/controller/suggestRecipe.ts]: Yeap, missed both. Commented

**Comment by DConnorFong** [comment]
Approved. Please resolve conflicts

**Pull request merged**